Makefile

```diff
@@ -51,7 +51,7 @@ lint-local: ## Use locally to run linters against Go code
 
 .PHONY: test
 test: ## Runs unit tests including checks for race conditions and returns coverage
-	go test -race -cover ./...
+	go test -race -cover -count=1 ./...
```

Comment:
According to the documentation, count should default to 1, so these changes should not be needed:
> 	-count n
>	      Run each test, benchmark, and fuzz seed n times (default 1).

Reply:
We had it setup in this way before and it was causing some of tests to have a `[cache]` flag - resulting in them passing locally, but not on the pipeline.